### PR TITLE
Add error handler that redirects to /error

### PR DESCRIPTION
**Pull request by mheisig**
Instead of dumping JSON errors to the page, this will redirect to a `/error?msg=Some error message` which the frontend will handle and display nicely.

### CODE REVIEW

**Review by BetaGrumm — APPROVED** [review]

File: lib/helpers/error-redirect.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+/**
+ * Use this method to render JSON error responses.
+ *
+ * @function
+ *
+ * @param {Object} res - Express http response.
+ * @param {Object} err - An error object.
+ */
+function errorRedirect(res, err) {
+  var message = 'Unknown error. Please contact support.';
```
Comment (thedrew12) [inline lib/helpers/error-redirect.js:12]: Very helpful error message. When would this message get sent to the redirect?

File: lib/helpers/error-redirect.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+  if (err) {
```
Comment (thedrew12) [inline lib/helpers/error-redirect.js:14]: Why are checking if there is an error? Shouldn't there always be an error?

File: lib/helpers/error-redirect.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
+'use strict';
```
Comment (thedrew12) [inline lib/helpers/error-redirect.js:1]: Is there a test for this function? Or does this fork of the branch not have any tests setup?

File: lib/helpers/error-redirect.js (hunk 1 above)
Comment (mheisig) [reply lib/helpers/error-redirect.js:12]: Only if no other message is included in the error object.

File: lib/helpers/error-redirect.js (hunk 2 above)
Comment (mheisig) [reply lib/helpers/error-redirect.js:14]: Copied this from the original jsonError function. I suppose it's possible to trigger it without an error object that includes a message?

File: lib/helpers/error-redirect.js (hunk 3 above)
Comment (mheisig) [reply lib/helpers/error-redirect.js:1]: `express-stormpath` doesn't have any tests for the `jsonError` function which is what the `error-redirect` is copied from. There's really nothing to test -- it just redirects.

**Review by thedrew12 — APPROVED** [review]

**Pull request merged**